crypto/crypto.ts

```diff
@@ -202,19 +210,17 @@ const stdCrypto: StdCrypto = ((x) => x)({
 
       const bytes = bufferSourceBytes(data);
 
-      if (FNV_ALGORITHMS.includes(name)) {
-        return fnv(name, bytes);
-      }
-
       // We delegate to WebCrypto whenever possible,
       if (
         // if the algorithm is supported by the WebCrypto standard,
-        (webCryptoDigestAlgorithms as readonly string[]).includes(name) &&
+        (WEB_CRYPTO_DIGEST_ALGORITHM_NAMES as readonly string[]).includes(
```

Comment:
```suggestion
        WEB_CRYPTO_DIGEST_ALGORITHM_NAMES.includes(
```
Since `WEB_CRYPTO_DIGEST_ALGORITHM_NAMES` is declared with `as const`.